src/main/java/survey/Question.java

```diff
@@ -1,6 +1,15 @@
 package survey;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
 public abstract class Question {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
     protected Integer id;
```

Comment:
Couple things:
1) shouldn't the type of the "id" field be a primitive
2) I feel like its better to use the a Long/long rather than a Integer